client-side/src/stories/header/header.jsx

```diff
@@ -90,31 +108,31 @@ function ResponsiveAppBar() {
               <Avatar>{getAvatarLetter()}</Avatar>
               </IconButton>
             </Tooltip>
-            <Menu
-              id="menu-appbar"
-              anchorEl={anchorElUser}
-              anchorOrigin={{
-                vertical: 'top',
-                horizontal: 'right',
-              }}
-              keepMounted
-              transformOrigin={{
-                vertical: 'top',
-                horizontal: 'right',
-              }}
-              open={Boolean(anchorElUser)}
-              onClose={handleCloseUserMenu}
-            >
-                        <LabTabs
-          nameOfClass="navbar-tabs"
-          text={['edit user profile','manage notifications']}
-          nav={['/editUserProfile','/manageNotifications'] }
-        />
-            </Menu>
+                {isSelectOpen && (
+                <Box sx={{ position: 'absolute', top: '100%', right: 0 }}>
+                  <Select
+                    onChange={(selectedValue) => selectFunctions(selectedValue)} // Directly pass the value
+                    className="primary"
+                    options={[
+                      { value: 1, text: "edit user profile", iconSrc: '/images/pencil.svg' },
```

Comment:
edit user command should appear only if the user is logged in

Reply:
ok